docs/terra/compute.rst

```diff
@@ -30,6 +30,19 @@ VirtualEnvCompute
 
 The :py:class:`terra.compute.virtualenv.Compute` is used to run services locally in a virtualenv. While terra itself runs in a virtualenv, the terra app has its own virtualenv, which typically exists in the container, but exists locally when using this compute. This compute does not use :ref:`settings-path-translation`.
 
+Debugging in the compute
+------------------------
+
+In order to debug in the exact compute environment, two environment were introduced to help:
+
+.. envvar:: TERRA_DEBUG_SERVICE
+
+Since there are many complicated steps that go into starting an environment for a service runner (i.e. starting a docker container with the right mounts and temporary settings file). Setting :envvar:`TERRA_DEBUG_SERVICE` will start the environment and run the command :envvar:`TERRA_DEBUG_SHELL` to allow you to debug in the actual environment with ease. :envvar:`TERRA_DEBUG_SERVICE` must be set to the service runner name or any class in it's class hierarchy (e.g. ``MyServiceRunner_docker``, ``MyServiceRunner``, or to stop on all runners ``object``).
```

Comment:
```suggestion
Since there are many complicated steps that go into starting an environment for a service runner (i.e. starting a docker container with the right mounts and temporary settings file). Setting :envvar:`TERRA_DEBUG_SERVICE` will start the environment and run the command :envvar:`TERRA_DEBUG_SHELL` to allow you to debug in the actual environment with ease. :envvar:`TERRA_DEBUG_SERVICE` must be set to the service runner name or any class in its class hierarchy (e.g. ``MyServiceRunner_docker``, ``MyServiceRunner``, or to stop on all runners ``object``).
```